### ISSUE TITLE
Accessibility stats and project/scenario names in cumulative opportunity plot buttons update before results are fetched

### ISSUE THREAD

**Issue by mattwigway** [conveyal/analysis-ui#409]
i.e. as soon as you change the comparison scenario, the text on the spectrogram/percentile curves selector changes, even if you haven't refreshed the results, creating a mislabeled chart.

**Comment by ansoncfit** [comment]
@trevorgerhardt I should have raised this when we were going through #795.  That PR grays out the percentile chart, but not the bars above it.  Maybe we should gray out the bars and remove the project: scenario names when the profile request is out of date.

**Comment by ansoncfit** [comment]
Another similar issue:
Fetch results and change the travel time slider.  Load a bookmark that was saved with a different travel time slider.  The origin marker and time slider will change, as will the blue/red accessibility statistic progressbars.  This might lead users to believe they are seeing updated accessibility statistics reflecting the new time cutoff and origin.  But what they are actually seeing reflects the new time cutoff and the old origin.

The accessibility statistic progress bars should be grayed out when isochrones are being fetched or the profile request is out of date.  I'd also suggest removing the project/scenario names from the progress bars and buttons below them when this is the case.
